geoip2/records.py

```diff
@@ -684,6 +684,20 @@ class Traits(Record):
 
       :type: unicode
 
+    .. attribute:: static_ip_score
+
+      An indicator of how static or dynamic an IP address is. The value ranges
+      from 0 to 99.99 with higher values meaning a greater static association.
+      For example, many IP addresses with a user_type of cellular have a
+      lifetime under one. Static Cable/DSL IPs typically have a lifetime above
+      thirty.
+
+      This indicator can be useful for deciding whether an IP address represents
+      the same user over time. This attribute is only available from GeoIP2
+      Precision Insights.
+
+      :type: decimal
```

Comment:
Also perhaps type float would be better here? That looks like what we use elsewhere.